tracee-ebpf/Makefile

```diff
@@ -1,5 +1,8 @@
 .PHONY: all
-all: build bpf
+all:
+	$(MAKE) bpf
+	$(MAKE) btfhub
+	$(MAKE) build
```

Comment:
sure, I needed ordering, while still allowing parallel compilation for inner steps of each rule, and this is the only way I could find to have it (by calling it recursively). Using pipes might help for other cases, not in this case. The $(MAKE), instead of just "make", is that it gets the MAKE options (like parallel jobs).

Reply:
I'm not a Makefile master (maybe @itaysk can help here), but I think this can be addressed by defining the dependencies of a target in the target itself (so btfhub can set bpf as a dependency)

Reply:
Yes, I would appreciate suggestions. I tried multiple different ways and wasn't able to find anything that satisfied this need of having **btfhub** to run in between **bpf** and **build** targets (while still allowing parallel jobs to each).

Reply:
see my comment about embedding https://github.com/aquasecurity/tracee/pull/1125#discussion_r746286358
it would simplify this along with other places

Reply:
anyway the "correct" solution IMO is to add the btfhub target as conditional dependency to the build target. but again this will all be simplified if we don't embed btfs in the binary, just build them for release